src/e2e-app/theme.scss

```diff
@@ -1,20 +1,21 @@
 @use '@angular/material' as mat;
-@use '@angular/material-experimental' as experimental;
 
 // Plus imports for other components in your app.
 
 // Include the common styles for Angular Material. We include this here so that you only
 // have to load a single css file for Angular Material in your app.
 // **Be sure that you only ever include this mixin once!**
 @include mat.core();
-@include experimental.all-mdc-component-typographies();
 
 // Define the default theme (same as the example above).
 $candy-app-primary: mat.define-palette(mat.$indigo-palette);
 $candy-app-accent: mat.define-palette(mat.$pink-palette, A200, A100, A400);
-$candy-app-theme: mat.define-light-theme($candy-app-primary, $candy-app-accent);
+$candy-app-theme: mat.define-light-theme((
+    color: (primary: $candy-app-primary, accent: $candy-app-accent),
```

Comment:
Indentation seems off here.

Reply:
what's off about it? it looks fine to me, and yarn format isn't complaining

Reply:
Maybe it's just me, but it seems like it's twice as indented as it should be. Either it's using 4 spaces or 2 tabs.